ibis/backends/duckdb/__init__.py

```diff
@@ -26,6 +27,50 @@ class _ColumnMetadata(NamedTuple):
     type: dt.DataType
 
 
+_register = RegexDispatcher("register")
```

Comment:
```suggestion
_register = RegexDispatcher("_register")
```
The tiniest of nits.